src/Carbon/Carbon.php

```diff
@@ -88,6 +88,7 @@ class Carbon extends DateTime implements JsonSerializable
     const MONTHS_PER_YEAR = 12;
     const MONTHS_PER_QUARTER = 3;
     const WEEKS_PER_YEAR = 52;
+    const DAYS_PER_MONTH = 30;
```

Comment:
I really don't like this part :(

Reply:
Indeed I would better imagine a configurable matrix, it's what I started to imagine for business time duration strings:
https://github.com/briannesbitt/Carbon/compare/master...kylekatarnls:factors

Also I think the steps are not obvious.
 Some users could prefer cascading 7 days => 4 weeks => 1 month, other would prefer to cascade directly from days to month.